packages/fetchai/agents/tac_participant_contract/aea-config.yaml

```diff
@@ -0,0 +1,44 @@
+agent_name: tac_participant
```

Comment:
please change name to `tac_participant_contract`, otherwise the fetch doesn't work